makemigrations.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+sys.path.insert(0, "tests")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+
+if __name__ == "__main__":
+    from django.core.management import execute_from_command_line
+    args = sys.argv
+    args.insert(1, "makemigrations")
+    args.insert(2, "django_ipam")
+
+execute_from_command_line(args)
```

Comment:
this is not needed, you can simply call `./manage.py makemigrations django_ipam`. You can keep the script on the filesystem to use it but please remove it from the repo using `git rm --cached <file>`